tests/testsuite/features2.rs

```diff
@@ -696,17 +696,57 @@ fn cyclical_dev_dep() {
         .build();
 
     // Old way unifies features.
-    p.cargo("run true").run();
+    p.cargo("run true")
+        .with_status(101)
+        .with_stderr(
+            "\
+error: failed to parse manifest at [..]
+
+Caused by:
+  a project can not depend on itself as a path dependency
+",
+        )
+        .run();
```

Comment:
Pretty sure this is not what this test was designed to check, but I have no idea on how to fix it properly.